daemons/ipa-kdb/ipa_kdb_mspac.c

```diff
@@ -80,6 +80,21 @@ static char *memberof_pac_attrs[] = {
 #define AUTHZ_DATA_TYPE_PAD "PAD"
 #define AUTHZ_DATA_TYPE_NONE "NONE"
 
+krb5_error_code filter_logon_info(krb5_context context,
+                                  TALLOC_CTX *memctx,
+                                  krb5_data realm,
+                                  struct PAC_LOGON_INFO_CTR *info);
+void get_authz_data_types(krb5_context context, krb5_db_entry *entry,
+                          bool *_with_pac, bool *_with_pad);
+krb5_error_code ipadb_adtrusts_fill_sid_blacklist(char **source_sid_blacklist,
+                                                  struct dom_sid **result_sids,
+                                                  int *result_length);
+krb5_error_code ipadb_adtrusts_fill_sid_blacklists(struct ipadb_adtrusts *adtrust,
+                                                   char **sid_blocklist_incoming,
+                                                   char **sid_blocklist_outgoing);
+krb5_error_code ipadb_mspac_check_trusted_domains(struct ipadb_context *ipactx);
+krb5_error_code ipadb_mspac_get_trusted_domains(struct ipadb_context *ipactx);
```

Comment:
These functions should just be made static - I don't see a need for a prototype.